README.rst

```diff
@@ -121,6 +121,24 @@ Prepare view sets to import/export via API
         resource_class = resources.BookResource
 
 
+If you use multiple or custom admin site, register required models to them
```

Comment:
Let's provide a link to the Django Documentation about multiple/custom admin site (https://docs.djangoproject.com/en/stable/ref/contrib/admin/#customizing-the-adminsite-class)